tests/common/common.h

```diff
@@ -511,7 +511,7 @@ struct image_access<3> {
  */
 template <typename T>
 struct check_type_existence {
-  check_type_existence() = default;
+  constexpr check_type_existence() = default;
 };
```

Comment:
I was thinking to just a function instead of a class:
```c++
template <typename T>
void constexpr check_type_existence() {
};
```

Reply:
Done in https://github.com/KhronosGroup/SYCL-CTS/pull/160/commits/cb1eff92dd00b931d5bc162b23124da7b4bb838a